sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -49,13 +49,13 @@
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist-esm dist-test typings *.tgz *.log samples/typescript/dist",
     "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
-    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/samples/typescript/src/",
+    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/keyvault-certificates/samples/typescript/src/",
```

Comment:
was it required to do this because of the test separation?

Reply:
No, this was because of the folder separation in https://github.com/Azure/azure-sdk-for-js/pull/8866 

I merged that PR without trying the live tests.